api/src/pcapi/core/offers/api.py

```diff
@@ -281,7 +281,7 @@ def update_offer(
                 offer.lastProvider,
             )
 
-    if offer.isFromProvider:
+    if offer.lastProvider is not None:
```

Comment:
Je ne comprends pas bien ces remplacements, dans la mesure où tu conserves la propriété `isFromProvider`.
Quel est le but ? « if offer is from provider » se lisait bien, non ?

Reply:
faire plaisir a mypy qui sinon disait qu'apre lastProvider pouvait être None (il eperait par la protection de isFromProvider)